Game/Source/ThirdPersonShooter/AIGym/ScaleTestableCharacter.cpp

```diff
@@ -7,6 +7,57 @@
 #include "Stats/Stats2.h"
 #include "SpatialNetDriver.h"
 #include "SpatialWorkerConnection.h"
+#include "Logging/LogMacros.h"
+
+#include "CoreMinimal.h"
+DEFINE_LOG_CATEGORY(LogBattleNonsense);
```

Comment:
I understand battle nonsense influenced this framework, but perhaps we should call it something a little more spatial? Likewise for the console variables.